elex/__init__.py

```diff
@@ -6,16 +6,22 @@
 from cachecontrol.caches import FileCache
 from elex.cachecontrol_heuristics import EtagOnlyCache
 
-__version__ = '2.4.3'
-_DEFAULT_CACHE_DIRECTORY = os.path.join(tempfile.gettempdir(), 'elex-cache')
+__version__ = "2.4.3"
+_DEFAULT_CACHE_DIRECTORY = os.path.join(tempfile.gettempdir(), "elex-cache")
 
-API_KEY = os.environ.get('AP_API_KEY', None)
-API_VERSION = os.environ.get('AP_API_VERSION', 'v2')
-BASE_URL = os.environ.get('AP_API_BASE_URL', 'http://api.ap.org/{0}'.format(API_VERSION))
-CACHE_DIRECTORY = os.environ.get('ELEX_CACHE_DIRECTORY', _DEFAULT_CACHE_DIRECTORY)
+API_KEY = os.environ.get("AP_API_KEY", None)
+
+API_VERSION = os.environ.get("AP_API_VERSION", "v3")
+BASE_URL = os.environ.get(
+    "AP_API_BASE_URL", "https://api.ap.org/{0}".format(API_VERSION)
+)
```

Comment:
@vutrobinnj Let's carry over the same changes to this README file over here too, please:

https://github.com/NationalJournal/elex/blob/a18d277ec9b1d649d55bb7794e74b9c786624772/docs/configuration.rst#L5-L11

Reply:
@cdornfeldnj Done